src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -259,6 +259,34 @@ def __init__(self, x, y, curve):
     def public_key(self, backend):
         return backend.load_elliptic_curve_public_numbers(self)
 
+    def encode_point(self):
+        # key_size is in bits. Convert to bytes and round up
+        byte_length = (self.curve.key_size + 7) // 8
+        return (
+            b'\x04' + utils.int_to_bytes(self.x, byte_length) +
+            utils.int_to_bytes(self.y, byte_length)
+        )
+
+    @classmethod
+    def from_encoded_point(cls, curve, data):
+        if not isinstance(curve, EllipticCurve):
+            raise TypeError("curve must be an EllipticCurve instance")
+
+        if data == b'\x00':
```

Comment:
I think you should just drop this branch, only `data.startswith(b'\x04')` and the `else`.